commcare/src/test/java/org/motechproject/commcare/web/ConfigControllerTest.java

```diff
@@ -79,10 +79,15 @@ public void shouldSetDefault() {
     public void testSaveSettings() throws BundleException, CommcareAuthenticationException, CommcareConnectionFailureException {
 
         Config config = ConfigsUtils.prepareConfigThree();
+        String oldName = config.getName();
+        controller.saveConfig(config, oldName);
 
-        controller.saveConfig(config);
+        verify(configService).saveConfig(config, oldName);
+
+        config.setName("updatedName");
```

Comment:
apart from fact that this test is rather redundant, I do not understand your changes. Why after veryfing you are setting name in config and then again use verify ? It doesn't make sense